SampleCode/README.md

```diff
@@ -1,30 +1,33 @@
 # Azure AI Document Intelligence Code Samples
-This repository contains example code snippets showing how Azure AI Document Intelligence can be used to get insights from documents.
 
-# Samples for REST API
-These code samples show common scenarios for using the REST API for Azure AI Document Intelligence in python and Jupyter notebooks.
-| File Name | Description |
-| --- | --- |
-| sample_get_all_text_lines.ipynb and sample_get_all_text_lines.py | Get all text lines from a document |
-| sample_get_all_tables.ipynb and sample_get_all_tables.py | Get all tables from a document |
-| ...| ... |
+> [!NOTE]
+> Form Recognizer is now **Azure AI Document Intelligence**!
+
+This repository contains example code snippets showing how Azure AI Document Intelligence can be used to get insights from documents.
 
 # Samples for Python SDK
-Refer to [this repo](https://github.com/Azure/azure-sdk-for-python/tree/azure-ai-formrecognizer_3.3.0b1/sdk/formrecognizer/azure-ai-formrecognizer/samples#samples-for-client-library-versions-320-and-later) for the samples using the latest SDK version.
+Refer to [this repo](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/documentintelligence/azure-ai-documentintelligence/samples) for the samples using the [latest SDK version](https://learn.microsoft.com/python/api/overview/azure/ai-documentintelligence-readme?view=azure-python-preview&preserve-view=true).
```

Comment:
remove the query params? `?view=azure-python-preview&preserve-view=true`

Reply:
removed.

Reply:
seems the query string parameters are still there? I'm ok with this URL though since this document mentioned "latest SDK version" so it's fine to use view=azure-python-preview

Reply:
remove &preserve-view=true, right?